docs/release-notes.asciidoc

```diff
@@ -11,6 +11,14 @@
 [[release-notes-7.9.0]]
 == 7.9.0
 
+[discrete]
+[[upgrade-notes]]
+=== Post upgrade requirement
+
+After upgrading, you need to <<enable-detections-ui, re-enable the Detections
+feature>>.
+Once enabled, all previously activated detection rules are activated.
```

Comment:
Technically activated rules will remain activated and continue to run at their scheduled interval even if the new permissions are not granted. These permissions are just for unlocking the UI (and should not affect existing activated rules).

Reply:
@spong Thanks for clarifying